lighting_import_attachment_smb/models/lighting_import_attachment.py

```diff
@@ -0,0 +1,15 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class LightingImportAttachment(models.Model):
+    _inherit = "lighting.import.attachment"
+
+    source = fields.Many2one(
+        comodel_name="lighting.import.attachment.backend",
+    )
```

Comment:
Define `required`, `tracking`, `cascade`, etc... and sql constraints (if any) at the very beginning of coding, if you don't there's a very high chances you forget.